Packages/ChineseNumbers/Sources/ChineseNumbers/SuzhouNumbers.swift

```diff
@@ -0,0 +1,73 @@
+import Foundation
+
+public class SuzhouNumbers: NSObject {
+
+    static private let verticalDigits: [Character: String] = [
+        "0": "〇", "1": "〡", "2": "〢", "3": "〣", "4": "〤",
+        "5": "〥", "6": "〦", "7": "〧", "8": "〨", "9": "〩"
+    ]
+    static private let horizontalDigits: [Character: String] = [
+        "1": "一", "2": "二", "3": "三"
```

Comment:
This is missing the mapping `"0": "〇"`, and so if you try to convert the number `1010` to Suzhou Number, it will crash the lookup at the current line 45: `horizontalDigits[c]!`.

Consider also add a unit test case for this (though we aren't running the ChineseNumbersTests as part of the CI workflow, are we)?

Reply:
Fixed.